adyenv6core/src/com/adyen/v6/util/TerminalAPIUtil.java

```diff
@@ -0,0 +1,213 @@
+package com.adyen.v6.util;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.List;
+import java.util.Map;
+import org.apache.log4j.Logger;
+import com.adyen.model.nexo.DocumentQualifierType;
+import com.adyen.model.nexo.ErrorConditionType;
+import com.adyen.model.nexo.OutputText;
+import com.adyen.model.nexo.PaymentReceipt;
+import com.adyen.model.nexo.ResultType;
+import com.adyen.model.terminal.TerminalAPIResponse;
+import com.google.common.base.Splitter;
+
+public class TerminalAPIUtil {
```

Comment:
any of these function important for other merchants? if so we can think of moving this into the java library.

Reply:
As we discussed before, if we keep it in java library then we may need to provide more functions to avoid "missing utility" tickets. So we kept it in hybris.

Reply:
ok